test/cypress/integration/settings/your-psp.cy.test.js

```diff
@@ -133,6 +159,46 @@ describe('Your PSP settings page', () => {
         expect(location.pathname).to.eq(`/your-psp`)
       })
     })
+
+    it('should not allow invalid 3DS Flex credentials to be saved', () => {
```

Comment:
This looks like it's re-testing https://github.com/alphagov/pay-selfservice/blob/master/test/cypress/integration/settings/your-psp.cy.test.js#L118-L127 but in a more thorough way. I wonder if we can delete https://github.com/alphagov/pay-selfservice/blob/master/test/cypress/integration/settings/your-psp.cy.test.js#L118-L127?

Reply:
So re-write https://github.com/alphagov/pay-selfservice/blob/master/test/cypress/integration/settings/your-psp.cy.test.js#L117 to be just a positive test.

Reply:
L118-L127 shows a user is entering the following details and submitted them and tests them to ensure that these values fail during validation process in self-service application before going to Connector for checking Worldpay 3DS Flex credentials and displays the error message.

```
Organisational unit ID: Invalid organisational unit ID
Issuer: Invalid issuer
JWT MAC key: Invalid JWT MAC key
```

If we remove these lines, we are going to skip testing `validate3dsFlexCredentials()` function and leading and trailing space removals. Are we okay to remove these lines to turn this test case into a positive test?

Reply:
Oh ok. Both tests test for invalid credentials but one test fails input format validation but this new one has valid formats but they are invalid in the sense they are not recognised by Worldpay.